pkg/utils/parse_arf_result.go

```diff
@@ -506,6 +593,43 @@ func handleDependencyAnnotation(u *unstructured.Unstructured, annotations map[st
 	return annotations
 }
 
+func handleValueUsed(ValuesList []string, annotations map[string]string) map[string]string {
+	trimedValuesList := ValuesList[:0]
+	for _, oriVal := range ValuesList {
+		re := regexp.MustCompile("([a-zA-Z]+(_[a-zA-Z]+)+)")
+		trimedValueMatch := re.FindStringSubmatch(oriVal)
+		if len(trimedValueMatch) > 1 {
+			trimedValuesList = append(trimedValuesList, trimedValueMatch[0])
+		}
+	}
+	annotations[compv1alpha1.RemediationValueUsedAnnotation] = strings.Join(trimedValuesList, ",")
+	return annotations
+}
+
+func handleNotFoundValue(NotFoundValues []string, annotations map[string]string) map[string]string {
+
+	annotations[compv1alpha1.RemediationUnsetValueAnnotation] = strings.Join(NotFoundValues, ",")
```

Comment:
No, it will not, only when a formated remediation is used, and there is no found value in the Result Config Map,
It is checked here, 

`if len(notFoundValueList) > 0 {
			annotations = handleNotFoundValue(notFoundValueList, annotations)
		}`

And it will only annotate the first not found value, since it is a parsing error, it will stop looking for values from the template. However, it will still show all used values, (the value that is going to be parsed will show in the value used)